Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -482,7 +482,7 @@ public class ProgramBuilder {
         }
         if type.Is(.function()) {
             let signature = type.signature ?? FunctionSignature(withParameterCount: Int.random(in: 2...5), hasRestParam: probability(0.1)) 
-            return definePlainFunction(withSignature: signature) { _ in
+            return definePlainFunction(withSignature: signature, isStrict: probability(0.2)) { _ in
```

Comment:
Is there a reason why the probability here is 0.2 but above it's 0.1? Maybe just use 0.1 everywhere for consistency?